enterprise/ha/src/test/java/jmx/HaBeanIT.java

```diff
@@ -68,7 +71,22 @@
     public static final ClusterRule clusterRule = new ClusterRule( HaBeanIT.class )
             .withInstanceSetting( setting( "jmx.port", STRING, (String) null ), intBase( 9912 ) )
             .withInstanceSetting( HaSettings.ha_server, stringWithIntBase( ":", 1136 ) )
-            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) );
+            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) )
+            .withStoreDirInitializer( new ClusterManager.StoreDirInitializer()
```

Comment:
Why is this recursive deletion especially important for this test? Why isn't it required in general for other tests?